versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+   three kubernetes clusters.
+
+2. You have set up the environment to install the KubeSlice Controller and the KubeSlice Worker. For more 
+   information, see [Prerequisites](https://kubeslice.io/documentation/open-source/1.3.0/category/prerequisites).
+
+3. Install [Kubectx](https://github.com/ahmetb/kubectx) to easily switch context between clusters.
+
+4. Kubeconfig files to access the Kubernetes clusters.
+
+
+## Configure Distributed MongoDB
+
+In this demonstration:
+- We will use the three Kubernetes clusters called `k8s-cluster-1`, `k8s-cluster-2`, and `k8s-cluster-3`.
+- Use the `cluster-1.config`, `cluster-2.config`, `cluster-3.config` kubeconfig files for the `k8s-cluster-1`, `k8s-cluster-2`, 
+  and `k8s-cluster-3` respectively.
+- Install KubeSlice using the `kubeslice-cli` tool. Use the topology file to install the KubeSlice Controller and the 
+  KubeSlice Worker in the k8s-cluster-1 cluster. Install the KubeSlice Worker in the `k8s-cluster-2` and `k8s-cluster-3` clusters 
+  using the same topology file.
+- Create a slice called `demo-slice` across three worker clusters and onboard the `mongodb` namespace.
+- Deploy the MongoDB Kubernetes Operator and Ops manager on the `k8s-cluster-1` cluster,  where the KubeSlice Controller is installed.
+- Deploy the MongoDB across the three clusters using MongoDBMulti.
+- Create a MongoDB replica set using MongoDBMultiCRD.
+- Create the ServiceExport. The service export will be created in all 3 clusters as each cluster will have one replica set.
+
+### Step 1: Clone the Examples Repo
+
+Clone the `examples` repo as it contains all the example YAML files in the 
+`examples/distributed-mongodb` directory. You can use these YAML files to configure MongoDB.
+
+Use the following command to clone the `examples` repo:
+
+```
+git clone https://github.com/kubeslice/examples.git
+```
+
+After cloning the repo, use the files from the `examples/distributed-mongodb` directory.
+
+### Step 2: Merge Kubeconfig Files
+
+1. Replace the `/path/to/kubeconfig/` path to with your local path to access the kubeconfig file.
+
+   ```
+   export KUBECONFIG=/path/to/kubeconfig/cluster-1.config:/path/to/kubeconfig/cluster-2.config:/path/to/kubeconfig/cluster-3.config
+   ```
+   ```
+   kubectl config view --flatten=true > merged.config
+   ```
+
+2. Verify the `merged.config` kubeconfig file using the following command:
+
+   ```
+   export KUBECONFIG=/path/to/kubeconfig/merged.config
+   ```
+
+   ```
+   kubectx
+   ```
+   
+   Expected Output
+
+   ```yaml
+   k8s-cluster-1 # kubeslice controller cluster & kubeslice worker cluster 1 & Mongodb central cluster & mongodb member cluster 1
+   k8s-cluster-2 # kubeslice worker cluster 2 & mongodb member cluster 2
+   k8s-cluster-3 # kubeslice worker cluster 3 & mongodb member cluster 3
+   ```
+
+
+### Step 3: Install KubeSlice
+
+To install KubeSlice using the `kubeslice-cli` tool:
```

Comment:
I have added downloading the  `kubeslice-cli` tool as a prerequisite. 

You can remove the steps related to the `kubeslice-cli` utility if you move it under prereq